File: MyAFBase/MyAFBase/Utils/ContactConfig.swift
import Foundation

/// Public contact channels for MyAFBase support and legal inquiries.
enum ContactConfig {
    static let supportEmail = "support@myafbase.com"
    static let legalEmail = "legal@myafbase.com"

    static let websiteURL = URL(string: "https://myafbase.com")!
    static let supportURL = URL(string: "https://myafbase.com/support")!
    static let privacyURL = URL(string: "https://myafbase.com/privacy")!
    static let userChoicesURL = URL(string: "https://myafbase.com/user-choices")!
    static let termsURL = URL(string: "https://myafbase.com/terms")!

    static var supportMailtoURL: URL? {
        URL(string: "mailto:\(supportEmail)")
    }

    static var legalMailtoURL: URL? {
        URL(string: "mailto:\(legalEmail)")
    }
}

File: android/app/src/main/java/com/ryanladuca/myafbase/utils/ContactConfig.kt
package com.ryanladuca.myafbase.utils

object ContactConfig {
    const val SUPPORT_EMAIL = "support@myafbase.com"
    const val LEGAL_EMAIL = "legal@myafbase.com"
    const val WEBSITE_URL = "https://myafbase.com"
    const val SUPPORT_URL = "https://myafbase.com/support"
    const val PRIVACY_URL = "https://myafbase.com/privacy"
    const val USER_CHOICES_URL = "https://myafbase.com/user-choices"
    const val TERMS_URL = "https://myafbase.com/terms"

    val supportMailtoUrl: String get() = "mailto:$SUPPORT_EMAIL"
    val legalMailtoUrl: String get() = "mailto:$LEGAL_EMAIL"
}
